tests/halide.rs

```diff
@@ -330,4 +335,132 @@ mod test {
             false,
         )
     }
+
+    // Uncomment to run the fancy 75% recipe.
```

Comment:
Can you add to the  comment explaining that this recipe can only run on leviathan and that's why it's commented out

Reply:
Done.